beacon-chain/rpc/eth/blob/handlers_test.go

```diff
@@ -238,6 +261,59 @@ func TestBlobs(t *testing.T) {
 		require.NoError(t, json.Unmarshal(writer.Body.Bytes(), resp))
 		require.Equal(t, len(resp.Data), 0)
 	})
+	t.Run("outside retention period returns 200 w/empty list ", func(t *testing.T) {
+		u := "http://foo.example/123"
```

Comment:
How come is the block outside of retention period when the retention period is set to 4096*32 slots?